build/cloud/ha/01-ha-prepare.yaml

```diff
@@ -0,0 +1,608 @@
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            objectType:
+              description: NodeSelector indicates the binding preferences between
+                devices and nodes. Refer to k8s.io/kubernetes/pkg/apis/core NodeSelector
+                for more details
+              type: string
+            objectName:
+              description: 'Required: The protocol configuration used to connect to
+                the device.'
+              type: string
+        status:
+          properties:
+            objectResourceVersion:
+              description: 'Required: DeviceModelRef is reference to the device model
+                            used as a template to create the device instance.'
+              type: string
+  version: v1alpha1
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: objectsyncs.reliablesyncs.kubeedge.io
+spec:
+  group: reliablesyncs.kubeedge.io
+  names:
+    kind: ObjectSync
+    plural: objectsyncs
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            objectType:
+              description: NodeSelector indicates the binding preferences between
+                devices and nodes. Refer to k8s.io/kubernetes/pkg/apis/core NodeSelector
+                for more details
+              type: string
+            objectName:
+              description: 'Required: The protocol configuration used to connect to
+                the device.'
+              type: string
+        status:
+          properties:
+            objectResourceVersion:
+              description: 'Required: DeviceModelRef is reference to the device model
+                           used as a template to create the device instance.'
+              type: string
+  version: v1alpha1
+
```

Comment:
Do we really need to cover crds here? Place them under build/crds is enough, WDYT?

Reply:
Here we can simplify the deployment steps and achieve one-click deployment.